src/runtime/pkg/containerd-shim-v2/create.go

```diff
@@ -70,7 +70,7 @@ func create(ctx context.Context, s *service, r *taskAPI.CreateTaskRequest) (*con
 	rootfs := filepath.Join(r.Bundle, "rootfs")
 
 	switch containerType {
-	case vc.PodSandbox:
+	case vc.PodSandbox, vc.SingleContainer:
```

Comment:
I'm wondering if a `Debug()` call could be useful somewhere around here to show the `containerType`.